### ISSUE TITLE
Exercise 2 oai - One cluster configuration 

### ISSUE THREAD

**Issue by chevalsumo** [nephio-project/nephio#737]
Hello, i successfully completed Exercise 2 - OAI without any issues. I would like to replicate it using a single cluster for all network components on a single cluster. However, the networks: vpc-cu-e1, vpc-cudu-f1, vpc-internal, vpc-internet, and vpc-ran require the CR Rawtopology, which is not necessary in my case since I only want a single cluster. I have tried several configurations without the CR Rawtopology or with only an empty topology, and even if the networks are in the "ready" state, I still encounter this error (Interface.n4 rpc error: code = Unknown desc = dynamic claim: no available routes based on the selector labels map[nephio.org/address-family:ipv4 nephio.org/cluster-name:core]) for all interfaces. I would like to ask you how to proceed, knowing that I do not want to use containerlab because I want my UE to access the interne.

![image](https://github.com/nephio-project/nephio/assets/116794393/f7836b1d-2bfe-4565-95b6-7f02d612adfe)
![image](https://github.com/nephio-project/nephio/assets/116794393/fc92bdd9-6790-4924-bb24-cf66b1c69649).

All operators are successfully  deployed,vlan created.

**Comment by arora-sagar** [comment]
Yes true you don't need it, it can be simpler. But for that, you need to remove all the networking CRs and change the configuration in the operator packages for example AMF and SMF N2 and N4 are configured for a particular topology like here --> https://github.com/OPENAIRINTERFACE/oai-packages/blob/r2/oai-cp-operators/operator/amf.yaml#L121 so we need to change it and move everything on `eth0`. This way all the NFs will use only one network

**Comment by chevalsumo** [comment]
Thank you for your response! I have two questions, however:

When you talk about CRs network, do you mean even the CR Interface of the NFs (not the operators), like (https://github.com/OPENAIRINTERFACE/oai-packages/blob/main/oai-amf/interface-n2.yaml) will be removed or just like the SBI interface?

The second question, if I understand correctly, the goal of the CRs Network is to have an abstraction over the infrastructure for all the NFs workflows. Is it possible to have more details on the implementation of the CR rawtopology to make it work in this case? Or in other cases where we don't use containerlab as an inter-cluster solution

**Comment by chevalsumo** [comment]
Hi,
So, I tried the proposed approach, I removed the CR network and modified the operators to point to eth0. Everything seems to be working except for the SMF, the operator is unable to instantiate the NF, I have the following error message from the logs:
<img width="684" alt="image" src="https://github.com/nephio-project/nephio/assets/116794393/8bd1a631-f392-4001-9e3a-2492e168d014">
Thank you for guiding me towards some ideas if you have any!
